src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1045,7 +1045,7 @@ Take a moment to preview the various utilities listed or view our comprehensive
     <value>release notes</value>
   </data>
   <data name="Oobe_ColorPicker_HowToUse.Text" xml:space="preserve">
-    <value>{Win} + {Ctrl} + {C} to open Color Picker.</value>
+    <value>{Win} + {Shift} + {C} to open Color Picker.</value>

```

Comment:
This is the only file that needs to be changes, all other files should be left untouched because they get updated by the internal localization process.

Reply:
Even the ones that use different words for the modifier keys? Like Dutch (Shift -> UMSHALT)?

Reply:
None of the files under `https://github.com/microsoft/PowerToys/tree/master/src/modules/launcher/PowerLauncher/loc` should be changed. Those files are updated by our internal localization process.

Reply:
Done, reverted all commits to the first one I made, before doing locales.